modules/manually-create-identity-access-management.adoc

```diff
@@ -160,6 +169,32 @@ spec:
     skipServiceCheck: true
 ----
 endif::google-cloud-platform[]
+ifdef::alibabacloud[]
+.Sample `CredentialsRequest` object
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cloud-credential-operator-ram-ro
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AlibabaCloudProviderSpec
+    statementEntries:
+    - action:
+      - ecs:CopySnapshot
+      - ecs:DeleteDisk
+      - ecs:DescribeInstanceAttribute
+      - ecs:DescribeInstances
+      effect: Allow
+      resource: '*'
+  secretRef:
+    namespace: cloud-credential-operator-ram-ro-creds
+    name: openshift-cloud-credential-operator
+----
+endif::alibabacloud[]
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz
Can we get an example `CredentailsRequest` object for Alibaba?

Reply:
https://github.com/openshift/cluster-image-registry-operator/blob/master/manifests/01-registry-credentials-request-alibaba.yaml